src/prefect/cli/command.py

```diff
@@ -0,0 +1,164 @@
+"""Tasks for interacting with shell commands"""
+
+import logging
+import os
+import sys
+import tempfile
+from typing import List, Optional, Union
+
+import anyio
+import typer
+from anyio.streams.text import TextReceiveStream
+from pydantic import VERSION as PYDANTIC_VERSION
+
+from prefect import flow
+from prefect.cli._types import PrefectTyper
+from prefect.cli.root import app
+from prefect.client.schemas.schedules import CronSchedule
+from prefect.logging import get_run_logger
+from prefect.runner.storage import ModuleFunction
+
+if PYDANTIC_VERSION.startswith("2."):
+    pass
+else:
+    pass
+
+command_app = PrefectTyper(name="command", help="Commands for working with commands.")
+app.add_typer(command_app)
+
+
+@flow
+async def shell_run_command(
```

Comment:
any reason not to [use `run_process`](https://github.com/PrefectHQ/prefect/blob/main/src/prefect/utilities/processutils.py#L239) here?